tests/base_hubspot.py

```diff
@@ -0,0 +1,150 @@
+import os
+import unittest
+from datetime import datetime as dt
+from datetime import timedelta
+
+import tap_tester.menagerie   as menagerie
+import tap_tester.connections as connections
+import tap_tester.runner      as runner
+from tap_tester.base_suite_tests.base_case import BaseCase
+from tap_tester import LOGGER
+
+
+class HubspotBaseCase(BaseCase):
+
+    # set the default start date which can be overridden in the tests.
+    start_date = BaseCase.timedelta_formatted(dt.utcnow(), delta=timedelta(days=-1))
+
+    EXTRA_FIELDS = {
+        'deals': {
+            # BUG_TDL-14993 | https://jira.talendforge.org/browse/TDL-14993
+            #                 Has an value of object with key 'value' and value 'Null'
+            'property_hs_date_entered_1258834',
+            'property_hs_time_in_example_stage1660743867503491_315775040'
+        },
```

Comment:
I think this is only used for the all_fields test.  This is duplicative of the bad keys below and isn't necessary as there are just specific examples.  We should move the bug info down to the method for removing the bad keys.

Reply:
Ah.. This just got carried over from the old test. 
So, we added these extra fields and removed the bad prefixes later.
Not needed here, will remove it.